build/periodic.sh

```diff
@@ -17,7 +17,7 @@ cloneRepos() {
 	REPOS=$(cat policy-grc-squad/main-branch-sync/repo.txt | grep policy | grep -v framework)
 	for repo in $REPOS; do
 		printf '%s\n' "Updating $repo ...."
-		git clone git@github.com:$repo.git $repo
+		git clone https://${GITHUB_USER}:${GITHUB_TOKEN}@github.com/$repo.git $repo
```

Comment:
Leaving the user and token off here will work since all the repos are public.

Reply:
I made the requested update. Thanks!